rollup.config.js

```diff
@@ -7,14 +8,24 @@ import { terser } from 'rollup-plugin-terser';
 import replace from 'rollup-plugin-replace';
 import json from 'rollup-plugin-json';
 
+if (process.env.VUE_VERSION !== 'vue2' && process.env.VUE_VERSION !== 'vue3') {
+  throw new Error(
+    'The environment variable VUE_VERSION (`vue2` | `vue3`) is required.'
+  );
+}
+
 const processEnv = conf => ({
   // parenthesis to avoid syntax errors in places where {} is interpreted as a block
   'process.env': `(${JSON.stringify(conf)})`,
 });
 
+const vuePlugin = process.env.VUE_VERSION === 'vue3' ? vueV3 : vueV2;
+const outputDir =
+  process.env.VUE_VERSION === 'vue3' ? 'dist/vue3' : 'dist/vue2';
```

Comment:
should the output dir for vue 2 stay the root?

Reply:
Yeah, I wanted to discuss with you about the output dirs.

With

```js
  "main": "dist/vue2/cjs",
  "module": "dist/vue2/es",
```

that config, for the vue2 users, it will be the same. So I think it's okay to have it under `dist/vue`.

Where should we have vue3 bundles?

For now, I'm importing like this:

```js
import ... from 'vue-instantsearch/dist/vue3/es';
```